### PR TITLE
#2 libft map get tests

### PR DESCRIPTION
**Pull request by ioannnikulin**
to build: `make fulltest`
to verify: make sure that `mapss: get in testing` shows up in the tests

### CODE REVIEW

**Comment by cr3mebrulee** [comment]
@ioannnikulin i see new PRs, can i take two days to concentrate on tree and check your PRs on Wednesday?

**Comment by cr3mebrulee** [comment]
works, `mapss: get in testing`

File: libft/tests/memory/ft_memcmp_test.c
Comment (cr3mebrulee) [inline libft/tests/memory/ft_memcmp_test.c:1]: line 26 - 32, do you want to keep commented block? i remember, you don't like it

File: libft/tests/map/mapss/ft_mapss_get_test.c [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	#ifdef DEBUG
+	printf("Starting %d tests.\n", SZ);
+	#endif
+	for (int i = START; i < SZ; i ++)
+	{
+		#ifdef DEBUG
+		printf("%i\n", i);
+		#endif
+		t_mapss *map = ft_mapss_gen(tests[i].content, tests[i].sz);
+		assert(map);
```
Comment (cr3mebrulee) [inline libft/tests/map/mapss/ft_mapss_get_test.c:60]: it's check, that map was created. it works for all cases. 
i propose to specify tests[12 - 14] and add check, that if map is empty than it initialized with 0.

 if (tests[i].sz == 0 || tests[i].content == NULL) 
 {
        assert(map->head == NULL && map->size == 0);
 }
 
i just tested, all tests passed. what do you think?

**Review by cr3mebrulee — CHANGES_REQUESTED** [review]

File: libft/tests/map/mapss/ft_mapss_get_test.c (hunk 1 above)
Comment (ioannnikulin) [reply libft/tests/map/mapss/ft_mapss_get_test.c:60]: I don't think it fits here, this is a PR on testing `get` operation, and you're talking about testing a generator... added this assertion anyway

Comment (cr3mebrulee) [reply libft/tests/map/mapss/ft_mapss_get_test.c:60]: Ok, i got your point, thanks

**Review by cr3mebrulee — APPROVED** [review]

**Pull request merged**